fastlane/lib/fastlane/actions/jira.rb

```diff
@@ -0,0 +1,80 @@
+module Fastlane
+  module Actions
+    class JiraAction < Action
+
+      def self.run(params)
+        require 'jira'
```

Comment:
This should be similar to the `cocoapods.rb` action:

```ruby
Actions.verify_gem!('jira')
require 'jir'
```
to show an appropriate error message if it can't be found